packages/pn-pa-webapp/src/pages/__test__/Dashboard.page.test.tsx

```diff
@@ -61,11 +54,7 @@ describe('Dashboard Page', async () => {
   it('Dashboard without notifications, clicks on new notification inside empty state', async () => {
     mock
       .onGet(
-        NOTIFICATIONS_LIST({
-          startDate: formatToTimezoneString(tenYearsAgo),
-          endDate: formatToTimezoneString(today),
-          size: 10,
-        })
+        '/bff/v1/notifications/sent?startDate=2014-05-03T00%3A00%3A00.000Z&endDate=2024-05-03T23%3A59%3A59.999Z&recipientId=&status=&iunMatch=&size=10'
```

Comment:
remove empty query parameters here and for all the other tests

Reply:
Unfortunately, this would cause a fail for the test because the real request returns the path with all params included.

Reply:
I tried and it worked

Reply:
It wasn't working at the beginning, maybe due to a bad encoding while copy-pasting